DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -59,7 +60,7 @@ public function load(array $configs, ContainerBuilder $container)
             $config['deployment_names'] = array_values(array_filter(explode(';', $config['application_name'])));
         }
 
-        $container->getDefinition('ekino.new_relic')
+        $container->getDefinition(Config::class)
```

Comment:
Should we make this change to more services?

Reply:
This is the new way to declare services. I'm :+1:

Reply:
I added: https://github.com/ekino/EkinoNewRelicBundle/issues/160